numpy/f2py/src/fortranobject.c

```diff
@@ -312,13 +312,11 @@ fortran_getattr(PyFortranObject *fp, char *name) {
             return NULL;
         return cobj;
     }
-    if (1) {
```

Comment:
This is [from 10 years ago](https://github.com/numpy/numpy/commit/4c1ae336a15e4f701848fac12a521ba67bf784d3#diff-2f941876fe4226986f07907b449a11b0R256) and I assume was an attempt to if guard it behind PY3K at that time

Reply:
It's a trick to deal with C89 declaration rules